diff --git a/spacewalk-usix.spec b/spacewalk-usix.spec
index 77f5ce4..0266659 100644
--- a/spacewalk-usix.spec
+++ b/spacewalk-usix.spec
@@ -11,7 +11,7 @@
 
 Name:	    spacewalk-usix
 Version:	2.8.1
-Release:	1%{?dist}
+Release:	2%{?dist}
 Summary:	Spacewalk server and client nano six library
 
 License:	GPLv2
@@ -86,6 +86,9 @@ cp $RPM_BUILD_ROOT%{pythonrhnroot}/common/usix.py $RPM_BUILD_ROOT%{python3rhnroo
 %endif
 
 %changelog
+* Wed Feb 28 2018 Tomas Kasparek <tkasparek@redhat.com> 2.8.1-2
+- rebuild for rhel8
+
 * Fri Feb 09 2018 Michael Mraka <michael.mraka@redhat.com> 2.8.1-1
 - remove install/clean section initial cleanup
 - removed Group from specfile
