diff --git a/spacewalk-usix.spec b/spacewalk-usix.spec
index 2bf8720..1b550d5 100644
--- a/spacewalk-usix.spec
+++ b/spacewalk-usix.spec
@@ -18,7 +18,7 @@
 
 Name:	    spacewalk-usix
 Version:	2.8.1
-Release:	4%{?dist}
+Release:	5%{?dist}
 Summary:	Spacewalk server and client nano six library
 
 License:	GPLv2
@@ -33,16 +33,6 @@ Obsoletes: spacewalk-backend-usix < 2.8
 %description
 Library for writing code that runs on Python 2 and 3
 
-%package -n python2-%{name}
-Summary: Spacewalk client micro six library
-Provides: python2-spacewalk-backend-usix = %{version}-%{release}
-Obsoletes: python2-spacewalk-backend-usix < 2.8
-BuildRequires: python-devel
-
-%description -n python2-%{name}
-Library for writing code that runs on Python 2 and 3
-
-%if 0%{?build_py3}
 %package -n python3-%{name}
 Summary: Spacewalk client micro six library
 Provides: python3-spacewalk-backend-usix = %{version}-%{release}
@@ -52,8 +42,6 @@ BuildRequires: python3-devel
 %description -n python3-%{name}
 Library for writing code that runs on Python 2 and 3
 
-%endif
-
 %prep
 %setup -q
 
@@ -62,35 +50,16 @@ Library for writing code that runs on Python 2 and 3
 %define debug_package %{nil}
 
 %install
-install -m 0755 -d $RPM_BUILD_ROOT%{pythonrhnroot}/common
-install -m 0644 __init__.py $RPM_BUILD_ROOT%{pythonrhnroot}/__init__.py
-install -m 0644 common/__init__.py $RPM_BUILD_ROOT%{pythonrhnroot}/common/__init__.py
-install -m 0644 common/usix.py* $RPM_BUILD_ROOT%{pythonrhnroot}/common/usix.py
-
-%if 0%{?build_py3}
-install -d $RPM_BUILD_ROOT%{python3rhnroot}/common
-cp $RPM_BUILD_ROOT%{pythonrhnroot}/__init__.py $RPM_BUILD_ROOT%{python3rhnroot}
-cp $RPM_BUILD_ROOT%{pythonrhnroot}/common/__init__.py $RPM_BUILD_ROOT%{python3rhnroot}/common
-cp $RPM_BUILD_ROOT%{pythonrhnroot}/common/usix.py $RPM_BUILD_ROOT%{python3rhnroot}/common
-%endif
+install -m 0755 -d $RPM_BUILD_ROOT%{python3rhnroot}/common
+install -m 0644 __init__.py $RPM_BUILD_ROOT%{python3rhnroot}/__init__.py
+install -m 0644 common/__init__.py $RPM_BUILD_ROOT%{python3rhnroot}/common/__init__.py
+install -m 0644 common/usix.py* $RPM_BUILD_ROOT%{python3rhnroot}/common/usix.py
+
 
 %clean
 
 %files
 
-%files -n python2-%{name}
-%dir %{pythonrhnroot}
-%dir %{pythonrhnroot}/common
-%{pythonrhnroot}/__init__.py
-%{pythonrhnroot}/common/__init__.py
-%{pythonrhnroot}/common/usix.py*
-%exclude %{pythonrhnroot}/__init__.pyc
-%exclude %{pythonrhnroot}/__init__.pyo
-%exclude %{pythonrhnroot}/common/__init__.pyc
-%exclude %{pythonrhnroot}/common/__init__.pyo
-
-%if 0%{?build_py3}
-
 %files -n python3-%{name}
 %dir %{python3rhnroot}
 %dir %{python3rhnroot}/common
@@ -100,9 +69,11 @@ cp $RPM_BUILD_ROOT%{pythonrhnroot}/common/usix.py $RPM_BUILD_ROOT%{python3rhnroo
 %{python3rhnroot}/common/__pycache__/*
 %exclude %{python3rhnroot}/__pycache__/*
 %exclude %{python3rhnroot}/common/__pycache__/__init__.*
-%endif
 
 %changelog
+* Mon Apr 16 2018 Tomas Kasparek <tkasparek@redhat.com> 2.8.1-5
+- build spacewalk-usix just on python3 (tkasparek@redhat.com)
+
 * Thu Mar 01 2018 Tomas Kasparek <tkasparek@redhat.com> 2.8.1-4
 - empty %%files section is required for a package to build as a metapackage
   (tkasparek@redhat.com)
